client/src/constants.js

```diff
@@ -8,6 +8,7 @@ import {
   MdVerifiedUser,
   MdAccountBalance,
   MdGavel,
+  MdRowing,
```

Comment:
I think a hammer is better for indicating the production page. What do you think?

![image](https://user-images.githubusercontent.com/60051944/114125469-8b1ede00-98c4-11eb-9739-a50a2d320b1e.png)

Reply:
![image](https://user-images.githubusercontent.com/43047923/114125867-46477700-98c5-11eb-9865-79e00c2c8daf.png)

Reply:
@Mackbellemore almost got away with it...

Reply:
@DavidLiang01 i totally agree 👍

Reply:
![image](https://user-images.githubusercontent.com/4203037/114125975-873f8b80-98c5-11eb-9db4-82ec31fb9a94.png)

How about this one? To stay with material design icons

Reply:
Both look like a great options.